pkg/drivers/powerstore_test.go

```diff
@@ -95,43 +95,64 @@ var (
 			name: "update HBNFS values for Node",
 			yamlString: `
 			- name: X_CSI_NFS_EXPORT_DIRECTORY
-              value: "<X_CSI_NFS_EXPORT_DIRECTORY>"
-            - name: X_CSI_NFS_CLIENT_PORT
-              value: "<X_CSI_NFS_CLIENT_PORT>"
-            - name: X_CSI_NFS_SERVER_PORT
-              value: "<X_CSI_NFS_SERVER_PORT>"`,
+		      value: "<X_CSI_NFS_EXPORT_DIRECTORY>"
+		    - name: X_CSI_NFS_CLIENT_PORT
+		      value: "<X_CSI_NFS_CLIENT_PORT>"
+		    - name: X_CSI_NFS_SERVER_PORT
+		      value: "<X_CSI_NFS_SERVER_PORT>"`,
 			csm:      csmForPowerStoreWithHBNFS("csm"),
 			ct:       powerStoreClient,
 			sec:      powerStoreSecret,
 			fileType: "Node",
 			expected: `
 			- name: X_CSI_NFS_EXPORT_DIRECTORY
-              value: "/var/lib/dell/myNfsExport"
-            - name: X_CSI_NFS_CLIENT_PORT
-              value: "2220"
-            - name: X_CSI_NFS_SERVER_PORT
-              value: "2221"`,
+		      value: "/var/lib/dell/myNfsExport"
```

Comment:
The default folder is myNfsExport here but somewhere down the line 151 is just nfs.. any reason for the difference?

Reply:
This one just formatting issues but it is testing for the detailed override of the parameter.

Further down is the addition of the minmal manifest version of this. If this parameter is omitted, it will default to `/var/lib/dell/nfs`.